src/DIRAC/Resources/Computing/SSHComputingElement.py

```diff
@@ -548,6 +533,8 @@ def __executeHostCommand(self, command, options, ssh=None, host=None):
                 index = output.index("============= Start output ===============")
                 output = output[index + 42 :]
             except Exception:
+                self.log.error("Invalid output from remote command", output)
+                self.log.exception()
```

Comment:
Any reason for not doing:
```suggestion
                self.log.exception("Invalid output from remote command", output)
```